pkg/cmd/admin/groups/examples/examples_test.go

```diff
@@ -0,0 +1,58 @@
+package examples
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/yaml"
+
+	"github.com/openshift/origin/pkg/cmd/server/api"
+	configapilatest "github.com/openshift/origin/pkg/cmd/server/api/latest"
+	"github.com/openshift/origin/pkg/cmd/server/api/validation"
+)
+
+func TestLDAPSyncConfigFixtures(t *testing.T) {
+	fixtures := []string{}
+
+	// build a list of common configurations for all schemas
+	schemas := []string{"rfc2307", "ad", "augmented-ad"}
+	for _, schema := range schemas {
+		fixtures = append(fixtures, schema+"/sync-config.yaml")
+		fixtures = append(fixtures, schema+"/sync-config-dn-everywhere.yaml")
+		fixtures = append(fixtures, schema+"/sync-config-partially-user-defined.yaml")
+		fixtures = append(fixtures, schema+"/sync-config-user-defined.yaml")
+	}
+
+	for _, fixture := range fixtures {
+		var config api.LDAPSyncConfig
+
+		yamlConfig, err := ioutil.ReadFile("./../../../../../test/extended/authentication/ldap/" + fixture)
+		if err != nil {
+			t.Errorf("could not read fixture at %q: %v", fixture, err)
+			continue
+		}
+
+		// we need to change the LDAP_SERVICE_IP field to a valid IP for this test
+		yamlConfig = bytes.Replace(yamlConfig, []byte("LDAP_SERVICE_IP"), []byte("0.0.0.0"), -1)
+
+		jsonConfig, err := yaml.ToJSON(yamlConfig)
+		if err != nil {
+			t.Errorf("could not convert YAML fixture at %q to JSON: %v", fixture, err)
+			continue
+		}
+
+		if err := runtime.DecodeInto(configapilatest.Codec, jsonConfig, &config); err != nil {
```

Comment:
I've confirmed this flow is identical to that in `oadm groups sync` as far as I can tell, so I don't quite understand why this is failing - the output of this `DecodeInto` is a `<nil>`-ed out object, which was the issue earlier with `oadm groups sync` when the wrong codec was being used. Here we infact use `configapilatest` Codec so this shouldn't be an issue?

@deads2k @liggitt thoughts?